src/diagram/diagram-api.tsx

```diff
@@ -90,6 +184,7 @@ export type Node = {
 
   items: EntityItem[];
```

Comment:
When we render a node, like a class or a profile, we render "a box". Inside the box we have attributes, or, in general, some items to render. The idea is that the items represent the content. Perhaps there is a better name to utilize?

Reply:
Ah ok, yeah maybe there is, but can't think of anything now. Maybe amplify the fact that it is for the rendering - `itemsToRender`, but that also probably isn't ideal since we can hide them and not render them (unless on hide they will be deleted from the `Node` and not present anymore)